src/components/BaseFormFields/BACDropdown.tsx

```diff
@@ -35,10 +35,12 @@ const BACDropdown = <T extends FieldValues>({
   fieldProps,
   customOnOptionSelect,
   customValue,
+  clearable,
 }: BaseFormField<T> & {
   fieldProps?: Partial<DropdownProps>;
   customOnOptionSelect?: onOptionSelectCallback<T>;
   customValue?: valueCallback<T>;
+  clearable?: boolean;
```

Comment:
You don't need to pass this down as it's own prop -- as the BACDropDown field passes down fieldProps already, so you just put the value there.  example fieldProps={{ clearable: true }}